src/background.js

```diff
@@ -3,7 +3,7 @@ import 'src/omnibar'
 
 
 function openOverview() {
-    browser.tabs.create({
+    browser.tabs.update({
```

Comment:
Maybe the best solution would be to check whether this is already an empty tab and based on that decide whether to create a new one or just open overview in current one?
Otherwise I can just remove that line from PR and leave it as it was.